scripts/linters/html_linter.py

```diff
@@ -209,6 +214,41 @@ def handle_data(self, data):
             elif data_line.startswith(ending_block):
                 self.indentation_level -= 1
 
+    def _check_space_between_attributes_and_values(
+            self, tag, attr, value, rendered_text, value_in_quotes):
+        """Checks if there are any spaces between attributes and their values.
```

Comment:
`between attribute and their value.` (This check only works for one attr and value, right?)

Reply:
done